src/transformers/__init__.py

```diff
@@ -767,6 +768,7 @@
     _import_structure["models.mobilenet_v1"].extend(["MobileNetV1FeatureExtractor", "MobileNetV1ImageProcessor"])
     _import_structure["models.mobilenet_v2"].extend(["MobileNetV2FeatureExtractor", "MobileNetV2ImageProcessor"])
     _import_structure["models.mobilevit"].extend(["MobileViTFeatureExtractor", "MobileViTImageProcessor"])
+    _import_structure["models.oneformer"].extend(["OneFormerFeatureExtractor", "OneFormerImageProcessor"])
```

Comment:
I think we can proceed with just creating an `ImageProcessor` in this PR, i.e. not creating a `FeatureExtractor` anymore.

cc @amyeroberts